draft-ietf-acme-acme.md

```diff
@@ -894,6 +904,7 @@ Content-Type: application/jose+json
     "url": "https://example.com/acme/new-reg"
   })
   "payload": base64url({
+    "external_secret": "XYZ123",
```

Comment:
On L637 `external_secret` is described as a "array of URIs". Should the example be in `[ ]` brackets? I.e.: `"external_secret": [ "XYZ123" ],` ?